pkg/subctl/cmd/join.go

```diff
@@ -69,6 +61,51 @@ var joinCmd = &cobra.Command{
 
 func joinSubmarinerCluster(subctlData *datafile.SubctlData) {
 
+	// Missing information
+	var qs = []*survey.Question{}
+
+	if len(clusterID) == 0 {
+		qs = append(qs, &survey.Question{
+			Name:   "clusterID",
+			Prompt: &survey.Input{Message: "What is your cluster ID?"},
+			Validate: func(val interface{}) error {
+				str, ok := val.(string)
+				if !ok {
+					return nil
+				}
+				return isValidClusterID(str)
+			},
+		})
+	}
+	if len(colorCodes) == 0 {
```

Comment:
can we also add a default color in the cobra cmdline variable?, that way we avoid asking the admin for this field. 

This is something rancher internal I think we even talked to chris once about removal... so i'd better not to ask the admin for unnecessary stuff. They could use labels/annotations on the clusters for that.

Reply:
Yes, indeed.

Reply:
ack thanks !! :)

Reply:
I’ve left the colour question here, but with #55 it won’t be asked unless the user explicitly specifies a blank colour code on the command line.